wordpress-mobile/WordPress-Android
# Navigate to privacy settings directly from GDPR popup

Fixes #18570

To test:
1. Launch app
2. Login with a user in a GRDPR jurisdiction
3. Tap `Settings` on the privacy consent bottom-sheet
4. **Expect** to see the privacy settings screen
   > **Note** Before, the button was opening the App Settings screen.

### Preview
https://github.com/wordpress-mobile/WordPress-Android/assets/4588074/2a3fb031-7e4d-4bd6-9380-4d77df69e120

## Regression Notes
1. Potential unintended areas of impact
   None, only adding minimum code to a enhance a feature which targeted `22.7`

6. What I did to test those areas of impact (or what existing automated tests I relied on)
   Manual testing

7. What automated tests I added (or what prevented me from doing so)
   None

PR submission checklist:
- [x] ~~UI Changes testing checklist~~: none, functional update
